bin/ebuild-helpers/fperms

```diff
@@ -20,12 +20,6 @@ for arg; do
 		args+=( "${arg}" )
 	else
 		args+=( "${ED%/}/${arg#/}" )
-		# remove the QA warning after 2024-12-31
-		if [[ ${arg:0:1} != / ]]; then
-			eqawarn "QA Notice: ${0##*/}: Path '${arg}' does not start with '/'."
```

Comment:
Should we make this a die?

Reply:
No, why? The path is relative to `${ED}` and therefore need not start with a slash. See https://bugs.gentoo.org/905223 for a longer explanation.

Reply:
Sorry, I'd completely forgot that context, my bad.